draft-ietf-tls-tls13.md

```diff
@@ -2939,6 +2939,14 @@ servers MUST process the client's ClientHello and then immediately
 send the ServerHello, rather than waiting for the client's
 EndOfEarlyData message.
 
+When using 0-RTT, clients MUST send its Finished message in response to
```

Comment:
"clients" -> "a client"
("clients" is plural, yet "its" & "message" are singular; should stay consistent)